### ISSUE TITLE
ios15 causes custom backButton to be replaced by native button on scroll

### ISSUE THREAD

**Issue by cspicuzza** [wix/react-native-navigation#7272]
## 🐛 Bug Report

After upgrading my simulator to iOS 15 several issues have surfaced. One has to do with using a custom button component. When scrolling, the button is replace by the native one.

## To Reproduce

(Write your steps here:)

1. Use a ios 15 simulator

2. Set RNN static props:
```
backButton: {
  id: Constants.TopBarButtonType.back,
  icon: props.icons.backButton.src,
  iconInsets: { left: -4 },
},
```

3. Navigate to the view and scroll up and down

## Expected behavior

I expected the custom component to display without changing

https://user-images.githubusercontent.com/4442172/134948668-78bf1e67-0e4f-4434-a7c2-ddecc1045451.mov

## Your Environment

- React Native Navigation version: 7.16.0
- React Native version: 0.63.3
- Platform(s) (iOS, Android, or both?): iOS
- Device info (Simulator/Device? OS version? Debug/Release?): iOS 15.0 / iPhone 13 Pro simulator

## Reproducible Demo

(Paste the link to an example repo and exact instructions to reproduce the issue.)

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✖️ &nbsp; Yes, I have the time, and I know how to start.
 - ✖️  &nbsp; Yes, I have the time, but I don't know how to start. I would need guidance.
 - ✖️  &nbsp; No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

Would love to contribute but native module PRs are out of my scope

**Comment by svbutko** [comment]
@cspicuzza, can you try the same behaviour but with RNN version `7.22.0`?

**Comment by svbutko** [comment]
Fixed, starting from `7.22.0` version
